File: src/Api/Models/AppSettings.cs
/*
* Copyright (C) Rodrigo Ferreira, All Rights Reserved
* Unauthorized copying of this file, via any medium is strictly prohibited
* Proprietary and confidential
*/

namespace FaceDetector.Api.Models;

public class AppSettings
{
    public string? ConnectionString { get; set; } = null;
    public string? DatabaseName { get; set; } = null;
    public string? ApiToken { get; set; } = null;
}

File: go-api/src/models/app_settings.go
/*
* Copyright (C) Rodrigo Ferreira, All Rights Reserved
* Unauthorized copying of this file, via any medium is strictly prohibited
* Proprietary and confidential
 */

package models

// AppSettings representa as configurações da aplicação
type AppSettings struct {
	ConnectionString string `json:"ConnectionString"`
	DatabaseName     string `json:"DatabaseName"`
	ApiToken         string `json:"ApiToken"`
}
